apps/condo/domains/billing/utils/constants.js

```diff
@@ -0,0 +1,17 @@
+const BILLING_CATEGORIES = [
+    '928c97ef-5289-4daa-b80e-4b9fed50c629',
```

Comment:
can we make some constants to name this ids?

Reply:
Removed this file